qubesbuilder/plugins/installer/Makefile

```diff
@@ -26,7 +26,7 @@ INSTALLER_DIR ?= $(PLUGINS_DIR)/installer
 SOURCES_DIR ?= $(BUILDER_DIR)/sources
 INSTALLER_KICKSTART ?= $(SOURCES_DIR)/qubes-release/conf/qubes-kickstart.cfg
 CREATEREPO := $(shell which createrepo_c createrepo 2>/dev/null |head -1)
-ISO_VERSION ?= $(shell date +%Y%m%d)
+ISO_VERSION ?= $(shell date -d @$SOURCE_DATE_EPOCH +%Y%m%d)
```

Comment:
This most likely needs a fallback too. But also, it's makefile, not shell script, so variables need $(), not just $